src/clj/garden/color.clj

```diff
@@ -330,6 +470,152 @@
         b (Math/round ^Double (* 0xff (+ b' m)))]
     (Rgb. r g b)))
 
+(def lab-constants
```

Comment:
This should probably be private. Also, either a docstring, or a short description (in comment form) above each of the keys explaining them would be useful.